tools/keepalived/keepalived/check/ipvswrapper.c

```diff
@@ -932,6 +932,7 @@ ipvs_blklst_vsg_cmd(int cmd, list vs_group, virtual_server_t * vs, blklst_addr_g
 static int
 ipvs_blklst_cmd(int cmd, list vs_group, virtual_server_t * vs)
 {
+        uint32_t ip_addr = 0;
```

Comment:
It's better to define `ip_addr` locally where it is used, because it is not used in other cases.